sycl/include/CL/sycl/INTEL/esimd/esimd.hpp

```diff
@@ -184,30 +184,26 @@ template <typename Ty, int N> class simd {
   // {/quote}
   //   * if not different, then auto should not be used
 #define DEF_BINOP(BINOP, OPASSIGN)                                             \
-  auto operator BINOP(const simd &RHS) const {                                 \
+  ESIMD_INLINE friend auto operator BINOP(const simd &x, const simd &y) {      \
```

Comment:
Please make parameter naming consistent with other friends (here and below):
```suggestion
  ESIMD_INLINE friend auto operator BINOP(const simd &Y, const simd &Y) {      \
```